### PR TITLE
Tm load test

### PR DESCRIPTION
**Pull request by puneetsingh166**
Its a new and updated version of the previous PR.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,15 +14,15 @@ require (
...
 	github.com/rakyll/statik v0.1.7
 	github.com/spf13/cast v1.3.1
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.8
+	github.com/tendermint/tendermint v0.34.14
 	github.com/tendermint/tm-db v0.6.4
 	github.com/testcontainers/testcontainers-go v0.11.1
-	go.opencensus.io v0.23.0 // indirect
```
Comment (puneetsingh166) [inline go.mod:25]: This all is happened because of ``` go mod tidy ```

File: Makefile [hunk 2]
```diff
@@ -102,7 +102,16 @@ lint-in-docker:
 .PHONY: format-in-docker
 format-in-docker:
 	make in-docker ARGS="make format"
+#########################################Build onomy load test binary#########################
 
+.PHONY: lint-tm
```
Comment (dzmitryhil) [inline Makefile]: you don't need to create a new lint just add tmload tag to the current lint

File: cmd/onomy-load-test/client.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+// ABCI app. The conversion to base64 will automatically be handled by the
+// loadtest package, so don't worry about that. Only return an error here if you
+// want to completely fail the entire load test operation.
+func (c *MyABCIAppClient) GenerateTx() ([]byte, error) {
+	txBytes, err := GenTx()
+	if err != nil {
+		// Handle the error!
+		return nil, err
+	}
+	return txBytes, nil
```
Comment (dzmitryhil) [inline cmd/onomy-load-test/client.go]: you can just keep "retrun GenTx()" here

File: go.mod (hunk 1 above)
Comment (dzmitryhil) [reply go.mod:25]: got it, it is because of new dependency which transitively provides updates

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

File: Makefile [hunk 4]
```diff
@@ -75,6 +75,7 @@ test-integration:
 .PHONY: lint
 lint:
 	golangci-lint -c dev/tools/.golangci.yml run --build-tags "integration"
+	golangci-lint -c dev/tools/.golangci.yml run --build-tags "tmload"
```
Comment (dzmitryhil) [inline Makefile]: no need for a new line update existing:
```
 golangci-lint -c dev/tools/.golangci.yml run --build-tags "integration tmload"
 ```

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

**Pull request merged**